docs/admin/administration.md

```diff
@@ -874,18 +874,18 @@ the system (this can be checked with `dmg system query -v`).
 ## Software Upgrade
 
 The DAOS v2.0 wire protocol and persistent layout is not compatible with
-previous DAOS versions and would require a reformat and all client and server
-nodes to be upgraded to a 2.x version.
+previous DAOS versions. It requires a reformat, and all client and server
+nodes need to be updated to a 2.x version.
 
 !!! warning
     Attempts to start DAOS v2.0 over a system formatted with a previous DAOS
     version will trigger a RAS event and cause all the engines to abort.
-    Similarly, a 2.0 DAOS client or engine will refuse to communicate with a
+    Similarly, a DAOS v2.0 client or engine will refuse to communicate with a
     peer that runs an incompatible version.
 
 DAOS v2.0 will maintain interoperability for both the wire protocol and
```

Comment:
(similar comment from the release/2.2 branch PR)
Since we are referring to the master (toward release 2.4) branch and now into the sequence of 2.x releases, something to consider may be to move this statement to the top of the section. i.e., for the most relevant scenario (a DAOS v2.2 installation being updated to DAOS v2.4), the straightforward (good) news is shown first. Then, we could include the DAOS v2.0 incompatibility with DAOS v1.x releases information next.

Reply:
good advice, but not the topic of this PR. let's land this and the re-arrangement of the interop statements can be done separately.